c_src/EiUtil.h

```diff
@@ -0,0 +1,212 @@
+#ifndef ELEVELDB_EIUTIL_H
+#define ELEVELDB_EIUTIL_H
+
+/**
+ * @file EiUtil.h
+ * 
+ * Tagged: Wed Sep  9 17:32:28 PDT 2015
+ * 
+ * @version: $Revision: $, $Date: $
+ * 
+ * @author /bin/bash: username: command not found
```

Comment:
Good catch -- yeah, these are auto-generated by some emacs macros I use to write C++ code.  They also put some outdated CVS-style tags in the header.  I've removed these from all the class files that had them, and replaced with a brief summary of the purpose of each class, creation date where available, and author email.